scripts/k8s-storage/sc-cephfs.yaml.in

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: k8s-storage-e2e-cephfs
+provisioner: rook-ceph.cephfs.csi.ceph.com
```

Comment:
Need to update the provisioner name to test with ceph-csi repo

Reply:
ah, right!